web/src/main/java/com/jos/dem/jmailer/controller/EmailerController.java

```diff
@@ -48,13 +51,19 @@
 public class EmailerController {
 
   private final EmailerService emailerService;
+  private final CommandValidator commandValidator;
 
   @Value("${token}")
   private String token;
 
   @Value("${email.redirect}")
   private String redirectUrl;
 
+  @InitBinder("formCommand")
+  private void initBinder(WebDataBinder binder) {
```

Comment:
Should that validation be in a service in `validateMessage`?

Reply:
That is an option. However, I prefer to use `@InitBinder` since it automatically validates a `DTO` coming to the controller. Here is a nice technical post about it: https://blog1.westagilelabs.com/how-to-use-initbinder-in-spring-mvc-ecb974a6884

Reply:
I like the suggestion; therefore, I created a new issue to move this business logic to a service: https://github.com/josdem/jmailer-spring-boot/issues/48